vector/src/main/java/im/vector/app/features/location/Config.kt

```diff
@@ -18,6 +18,7 @@ package im.vector.app.features.location
 
 const val MAP_BASE_URL = "https://api.maptiler.com/maps/streets/style.json"
 const val STATIC_MAP_BASE_URL = "https://api.maptiler.com/maps/basic/static/"
+const val USER_PIN_NAME = "USER_PIN_NAME"
```

Comment:
It should be more something like `const val GENERIC_PIN_NAME = "GENERIC_PIN_NAME"`, no?

Reply:
Not exactly, but I changed it to `DEFAULT_PIN_ID` for now. Refactoring will be needed during live location implementation.